### PR TITLE
Created the rentals controller with index, new actions, and private methods

### PR DESCRIPTION
**Pull request by MmeIsabelle**
Created action for the index and new.
Build the privates methods to set rental and listing and for the strong params.

### CODE REVIEW

**Comment by MmeIsabelle** [comment]
Ready for review @dylancherry98

File: app/controllers/rentals_controller.rb [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  end
+
+  def listing_rentals
+    #TODO
+  end
+
+  private
+
+  def set_rental
+    @rental = Rental.find(:id)
```
Comment (nafriedman) [inline app/controllers/rentals_controller.rb:31]: It is params[:id] here for line 31

Comment (MmeIsabelle) [reply app/controllers/rentals_controller.rb]: Fixed
Pushed a new commit

Comment (dylancherry98) [reply app/controllers/rentals_controller.rb]: New commit looks good!

**Comment by dylancherry98** [comment]
Looks good to me!

**Comment by nafriedman** [comment]
gonna merge this

**Pull request merged**